src/lib/enr/enr.spec.ts

```diff
@@ -8,30 +9,46 @@ import { bytesToHex, hexToBytes } from "../utils";
 import { ERR_INVALID_ID } from "./constants";
 import { ENR } from "./enr";
 import { createKeypairFromPeerId } from "./keypair";
+import { Waku2 } from "./waku2_codec";
 
 import { v4 } from "./index";
 
+use(chaibytes);
+
 describe("ENR", function () {
   describe("Txt codec", () => {
-    it("should encodeTxt and decodeTxt", async () => {
+    it.only("should encodeTxt and decodeTxt", async () => {
       const peerId = await PeerId.create({ keyType: "secp256k1" });
       const enr = ENR.createFromPeerId(peerId);
       const keypair = createKeypairFromPeerId(peerId);
       enr.setLocationMultiaddr(new Multiaddr("/ip4/18.223.219.100/udp/9000"));
       enr.multiaddrs = [
         new Multiaddr(
-          "/dns4/node-01.do-ams3.wakuv2.test.statusim.net/tcp/443/wss"
+          "/dns4/node1.do-ams.wakuv2.test.statusim.net/tcp/443/wss"
```

Comment:
this helped reduce the size of the ENR to under 300 bytes although I'm concerned about in the real world where we don't have control over this

Reply:
In the real world then:
- Operator can create several ENRs with different multiaddrs
- We can look into moving away from ENR if it's too limited.